### ISSUE TITLE
Accessibility Hover in Footer

### ISSUE THREAD

**Issue by yiwenxx** [pulibrary/tigerdata-app#1235]
### **Issue**

a. Accessibility Link hover behavior is not consistent with that in design.
b. The vertical alignment in the footer containing the Accessibility Link is not consistent with design.

**Issue a:**
- The current Accessibility link in the footer has a standalone hover effect, in which the entire text background changes to white and text color changes to black, not aligning with all other hover effects.

![Image](https://github.com/user-attachments/assets/0eb32672-c8b2-422a-8b8e-5bebb14682f9)

### **Implementation Notes**

- Keep the hover state the same as all other links. When hover, the content will be underlined.

![Image](https://github.com/user-attachments/assets/546aaf22-66a2-46d0-b66e-e7bee245efbb)

**Demo**

![Image](https://github.com/user-attachments/assets/ef0d0509-34b8-408f-8be9-250e09149f69)

**Issue b:**
The Accessibility Link, Copyright Text, and Princeton University Logo are not vertically centered.

![Image](https://github.com/user-attachments/assets/110187bd-38b8-4796-a3c2-89e0a186b16b)

Link to Figma: https://www.figma.com/design/rgzEYTpnGl6l68moRSVwYg/Phase-1-UI-Details?node-id=2536-4948&t=I3GXx7fYln7e6HP5-4

## Acceptance criteria
- [x] Hover state of accessibility link has underline
- [ ] Accessibility link, copyright notice, and Princeton logo are vertically aligned to match the design

**Comment by kelynch** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RDSS-61a4f1a12a399b001730f65a/issues/pulibrary/tigerdata-app/1235?planning-poker) with Zenhub @bess @JaymeeH @jrgriffiniii
